oneapp-server/package.json

```diff
@@ -40,7 +40,7 @@
     "graphql-middleware": "^4.0.2",
     "graphql-parse-resolve-info": "^4.9.0",
     "graphql-rate-limit": "^2.0.1",
-    "graphql-shield": "^7.4.1",
+    "graphql-shield": "^7.4.2",
```

Comment:
I'm assuming this is the result of something weird with dependabot because I bumped the version earlier in the week, haha.